src/components/btnGitHub/style.css

```diff
@@ -1,34 +0,0 @@
-.btn-outline {
```

Comment:
this file should be named the same as responsible *.js file.
File naming format convention is more usual like bnt-git-hub.js